sdk/python/kfp/dsl/__init__.py

```diff
@@ -17,5 +17,6 @@
 from ._pipeline import Pipeline, pipeline, get_pipeline_conf
 from ._container_op import ContainerOp
 from ._ops_group import OpsGroup, ExitHandler, Condition
-from ._component import python_component
+from ._component import python_component, component
+from ._types import Integer, String, Float, Bool, List, Dict, GCSPath, GCRPath, GCPRegion, GCPProjectID, LocalPath
```

Comment:
I wonder whether we should put types in `dsl.` namespace or have them in `dsl.types.`. Maybe the `dsl.types.` can be more convenient to the user as the `dsl.*` namespace will be smaller and more manageable as we add more and more types.

To put the types in the `dsl.types` namespace you can just rename `_types.py` to  `types.py`.

Reply:
+1. Better to expose those types under dsl.types module.

Reply:
good points. dsl.types sounds good to me.